app/state/queries/useFetchVelog.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+
+function todos({ status, page }) {
+  const result = useQuery({
+    queryKey: ['todos', { status, page }],
+    queryFn: fetchTodoList,
```

Comment:
`queryFn: (context: QueryFunctionContext) => Promise<TData>`

The function that the query will use to request data.